1209/600D/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: pid
+title: N93 Interface
+owner: Makdaam
```

Comment:
The capitalisation here needs to match the capitalisation of your org directory.